statsd.go

```diff
@@ -93,6 +93,24 @@ func Inc(name string) {
 	return
 }
 
+// Sample sends a counter with the given sample rate (between 0.0 - 1.0)
+func Sample(name string, rate float32){
+	if err := getConnection(); err != nil {
+		return
+	}
+
+	rate := "|@%f" + strconv.FormatFloat(rate, 'f', -1, 32)
+
+	conn.Write([]byte(name + ":1|c" + rate))
+
+	// Send a host suffixed stat too.
+	if AlsoAppendHost {
+		conn.Write([]byte(name + "." + host + ":1|c" + rate))
```

Comment:
are you sure this is the right format for this info?  looking at what I can see in another library  it looks like there should probably be an @ somewhere in there?

Reply:
The '@' is in the 'rate' string